src/categories/categories.service.ts

```diff
@@ -41,4 +44,18 @@ export class CategoriesService {
   deleteCategory(id: string): Promise<mongo.DeleteResult> {
     return this.categoryModel.deleteOne({ _id: id }).exec();
   }
+
+  async updateRank(rankBody: RankDto) {
+    const doc = rankBody;
+    const updates = [];
```

Comment:
Try to use map instead forEach in this case

Reply:
why does it better

Reply:
done